Add django, isa-l and libaom files (#138)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/isa-l.md

---

> name: isa-l

Author: ```suggestion
name: Isa-l
```

category: Storage
description: ISA-L is a collection of optimized low-level functions targeting storage applications. ISA-L includes Erasure codes, CRC, Raid, Compression, De-compression, igzip.

> download_url: https://github.com/intel/isa-l/archive/refs/tags/v2.26.0.tar.gz

Author: ```suggestion
download_url: https://github.com/intel/isa-l/tags
```

works_on_arm: true
supported_minimum_version:
    version_number: v2.26.0
    release_date: 26/03/2019

optional_info:
    homepage_url: https://www.intel.com/content/www/us/en/developer/tools/isa-l/overview.html
    support_caveats:

>     alternative_options: isa-l/README.md at v2.26.0 · intel/isa-l · GitHub

Author: remove this

    getting_started_resources:
        arm_content: 
        partner_content: 

>         official_docs: https://www.intel.com/content/www/us/en/developer/technical-library/overview.html?f:@stm_10184_en=%5BIntel%C2%AE%20Intelligent%20Storage%20Acceleration%20Library%20(ISA-L)%5D

Author: ```suggestion
        official_docs: https://github.com/intel/isa-l/blob/master/README.md
```
I think this is better

    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info:

Author: update other info

>     other_info: No arm64 specific release notes are available.Picked this version as found commits related to AARCH64 in this version, this version tested successfully. Also checked previous versions but got failure while testing.

Author: ```suggestion
    other_info: No arm64 specific release notes or binaries are available.  As per commits related to AARCH64 v2.26.0 version is minimum supported version. This version was built and tested successfully using tar file.
```
